packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/home/home.html

```diff
@@ -1,3 +1,139 @@
 <div>
-    home page
+    <div class="row my-2">
+        <!-- AI Pipeline-->
+        <div class="col-lg-8 col-12 mb-2">
+            <div class="row mb-4">
+                <div class="col">
+                    <ai-pipeline
```

Comment:
**nitpick:** Prefix the directives with `pci-project` to avoid naming collision.
```suggestion
                    <pci-project-ai-pipeline
```

Reply:
Changed: https://github.com/ovh/manager/pull/8172/commits/ede67bf06474e79c5ebd56df9418fdc3266b343d